src/styles/main.scss

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
 }
+
+$grey: #eee;
+$square-side: 100px;
+$calendar-gap: 1px;
+$calendar-day-border: 1px;
+$calendar-day-space: $square-side + $calendar-gap;
+$calendar-padding: 10px;
+$calendar-width: ($square-side + $calendar-gap) * 7;
+
+.calendar {
+  width: $calendar-width;
+  margin: 0 auto;
```

Comment:
The calendar should be centered both horizontally and vertically. Maybe try to add display flex to parent, which is body

Reply:
> The calendar should be centered both horizontally and vertically. Maybe try to add display flex to parent, which is body

I was set 100vh for body, and use flex alignment